docs/crio.conf.5.md

```diff
@@ -279,6 +279,8 @@ CRI-O reads its configured registries defaults from the system wide containers-r
 **big_files_temporary_dir**=""
   Path to the temporary directory to use for storing big files, used to store image blobs and data streams related to containers image management.
 
+**separate-pull-cgroup**=false
+  If true, image pulls will happen in a new cgroup. It is currently supported only with the systemd cgroup manager.
```

Comment:
Can you combine the messaging in this man and crio.8.md?  Perhaps in both places:
```suggestion
  If true, image pulls will happen in a new cgroup. It is currently supported only with the systemd cgroup manager.  (default: false)
```
Just based on this, there's no discussion about why one might want to use a new cgroup.  Is it covered elsewhere in this man page?  If not, we should add it either here or somewhere on the page.

Reply:
we are not specifying any default value in `crio.8.md` so wouldn't it be better to be conformant with the rest of the file?

Reply:
@giuseppe being conformant with crio.8.md and leaving the default out there is fine by me.  However, it would be best to list or not list defaults in all man pages and be consistent.  But that's a task for another day.